include/openssl/ssl3.h

```diff
@@ -25,6 +25,9 @@
 extern "C" {
 #endif
 
+/* Flag used on OpenSSL ciphersuite ids to indicate they are for SSLv3+ */
+# define SSL3_CK_CIPHERSUITE_FLAG                0x03000000
```

Comment:
That's what I was saying "I can't" to? We can't change the cipher suite id values because they are in a public header file. (BTW there are no sslv2 ciphersuites...they all have the flag)